### PR TITLE
Fix various imports

### PR DESCRIPTION
**Pull request by sgugger**
# What does this PR do?

While building the v2 of the test fetcher, I discovered (I mean the util discovered) that some imports in the source code are wrong. This PR fixes all of them.

### CODE REVIEW

File: src/transformers/models/data2vec/convert_data2vec_text_original_pytorch_checkpoint_to_pytorch.py [hunk 1]
```diff
@@ -31,11 +31,11 @@
     BertOutput,
     BertSelfAttention,
     BertSelfOutput,
+    Data2VecTextModel,
```
Comment (ydshieh) [inline src/transformers/models/data2vec/convert_data2vec_text_original_pytorch_checkpoint_to_pytorch.py:34]: I don't think `transformers.models.bert.modeling_bert` has `Data2VecTextModel`.

**Review by ydshieh — APPROVED** [review]
Thank you @sgugger ! I have a question regarding `Data2VecTextModel` in `src/transformers/models/data2vec/convert_data2vec_text_original_pytorch_checkpoint_to_pytorch.py`, but all other changes are good for me.

It seems test fetcher v2 is super strong to detect these kinds of mistake(s). Eager to see it's merged into `main` 🔥 .

File: src/transformers/models/data2vec/convert_data2vec_text_original_pytorch_checkpoint_to_pytorch.py (hunk 1 above)
Comment (sgugger) [reply src/transformers/models/data2vec/convert_data2vec_text_original_pytorch_checkpoint_to_pytorch.py:34]: Woopsie, wanted to put that in the line above. Thanks for the catch!

**Review by LysandreJik — APPROVED** [review]
Nice catch! Impressive the amount of imports that were wrong

**Pull request merged**